README.md

```diff
@@ -287,13 +287,13 @@ The method form takes parameters
 `auth(username, password, sendImmediately, bearer)`.
 
 `sendImmediately` defaults to `true`, which causes a basic or bearer
-authentication header to be sent.  If `sendImmediately` is `false`, then
```

Comment:
In the future, address these unrelated changes in a separate PR. It makes the review smaller/easier.